Hash ode map json deduplication (#11)
CDOT-CV/jpo-deduplicator: README.md

### OdeMapJson
- Two messages within a 1 hour time window 
- Two messages have the same intersection ID
- The Messages have the same hash values after remove the `ASN1`, `OdeReceivedAt` and  `MOY` fields

### ProcessedMap and ProcessedMapWKT
- Two messages within a 1 hour time window

> - Two messages have the same hash values after factoring out the `ASN1`, `odeReceivedAt` and `timeStamp` fields

Reviewer: nitpick
```suggestion
- Two messages are the same after factoring out the `ASN1`, `odeReceivedAt` and `timeStamp` fields
```
